main.py

```diff
@@ -2,6 +2,25 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    [Genre.objects.create(name=genre) for genre in genres]
```

Comment:
use regular for loops to iterate over the values

Reply:
okay, will fix it 😄